files/base.rb

```diff
@@ -67,11 +67,15 @@ def tip
     @event['check']['tip']
   end
 
-  def description(maxlen=100000)
-    description = @event['check']['notification']
+  def client_display_name
     client_display_name = @event['client']['tags']['Display Name'] rescue nil
     client_display_name = @event['client']['name'] if
       client_display_name.nil? || client_display_name.empty?
+    client_display_name
+  end
+
+  def description(maxlen=100000)
+    description = @event['check']['notification']
     description ||= [client_display_name, @event['check']['name'], uncolorize(@event['check']['output'])].join(' : ')
```

Comment:
It is in the description, it's somewhat confusing diff formatting.

I only took how client_display_name var was defined and put it into a separate method so that both description and full_description could access it.

So description itself has not changed.